docs/kytConfig.md

```diff
@@ -37,6 +37,8 @@ modifyWebpackConfig: (baseConfig, options) => {
 }
 ```
 
+> **NOTE:** [babel-polyfill](https://babeljs.io/docs/usage/polyfill/) is included in the Webpack configs' entry.main arrays by default. If you use `modifyWebpackConfig()` to override the entry properties and wish to keep using the polyfill, the new entries should go after the polyfill.
```

Comment:
That sounds great. I'll make the change and push it up when I have the chance. 

Also, I need to investigate why the e2e tests are failing for this PR. Permissions denied on the starter kyt repos. I'll take a look tonight. Thanks @tizmagik